admin/timescale/create_tables.sql

```diff
@@ -256,6 +256,12 @@ CREATE TABLE similarity.artist_credit_mbids (
     score INT NOT NULL
 );
 
+CREATE TABLE similarity.overhyped_artists (
+    id                      SERIAL,
```

Comment:
Use `INTEGER GENERATED ALWAYS AS IDENTITY NOT NULL` instead of `SERIAL`

Reply:
Done and done. Updated in production.